components/GithubUsernameForm.tsx

```
@@ -71,13 +215,109 @@ export default function GithubUsernameForm() {
+                    {category}
+                  </h3>
+                  <div className="flex items-center gap-2">
+                    <button
+                      onClick={() => handleCreateGitHubList(category, repos)}
+                      className="bg-green-100 dark:bg-green-900/30 text-green-800 dark:text-green-200 text-xs font-medium px-2 py-1 rounded-full hover:bg-green-200 dark:hover:bg-green-900/50 flex items-center transition-colors"
+                      title="Save List"
+                      disabled={isCreatingList}
+                    >
+                      {isCreatingList && creatingListCategory === category ? (
+                        <span className="flex items-center">
+                          <svg className="animate-spin h-3 w-3 mr-1" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)